packages/app_store/apple_app_store/.gitignore

```diff
@@ -0,0 +1,26 @@
+# Miscellaneous
```

Comment:
I think I mentioned it a while back in another review already. I'd rather have one global `.gitignore` with some local overrides then a lot of them all over the place